src/pages/home/_hero.js

```diff
@@ -114,8 +114,8 @@ const ImageWrapper = styled(Box)`
 const Hero = ({ is_ppc }) => {
     const data = useStaticQuery(query)
     const typewriter_text = !is_ppc
-        ? localize('Trade forex, commodities, synthetic and stock indices')
-        : localize('Trade forex, commodities, and stock indices')
+        ? localize('Trade forex, commodities, synthetic and stocks & indices')
```

Comment:
I have a feeling something needs to be changed here 😅 Shouldn't it be `synthetic indices and stocks & indices`?

Reply:
Discussed this with lipika and it should be `Trade forex, commodities, synthetics, and stocks & indices` instead